content/01-install/01a-install-client-windows.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Install Client on Windows
+metaTitle: Install Client on Windows
+---
+
+This section outlines how to install the Mantis client on your Windows machine.
+
+Follow these instructions to install the Mantis client or go [here for instructions to install the Mantis wallet](/install/install-wallet).
+
+
+## Requirements
+
+These are the prerequisites for installing Mantis:
+* A Java Virtual Machine (JVM) with version 1.8.x.
+    To check the JVM version, use `java -version`.
+    To install Java, follow [these instructions](https://java.com/en/download/help/download_options.html).
+    (Note that the Mantis team has not tested the client with JVM 1.9 or later versions)
+* 4G of memory (RAM)
+    Additional RAM is needed for the DAG file _if mining_ is enabled.
+* 250GB of disk space to ensure fast sync of the node. You will need more than that for the future growth of the chain. An SSD of at least 500G is recommended for full sync.
```

Comment:
seems like a lot of space is required, is this double checked?

Reply:
It was in the original document. Considering it has to hold the entire blockchain it does not seem excessive to me.

Reply:
yeah, you are probably right